nifi-registry-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -287,34 +290,30 @@ The CompositeConfigurableUserGroupProvider will provide support for retrieving u
 * Configurable User Group Provider - A configurable user group provider.
 * User Group Provider - The identifier of user group providers to load from. The name of each property must be unique, for example: "User Group Provider A", "User Group Provider B", "User Group Provider C" or "User Group Provider 1", "User Group Provider 2", "User Group Provider 3"
 
+After you have configured a UserGroupProvider, you configure an AccessPolicyProvider that will control Access Policies for the identities in the UserGroupProvider.
```

Comment:
To be consistent with rest of doc, add "must" in "After you have configured...., you must configure an AccessPolicyProvider..."